homeassistant/components/mysensors.py

```diff
@@ -125,7 +124,8 @@ def mysensors_callback(gateway, node_id):
         for child in gateway.sensors[node_id].children.values():
             for value_type in child.values.keys():
                 key = node_id, child.id, value_type
-                if child.type not in s_types or value_type not in v_types:
+                if child.type not in map_sv_types or \
```

Comment:
Yes, I came to the same conclusion after a nights sleep. I'll change to a regular dict of lists. That way, I don't have to worry about mutating the map by accident, eg when checking membership.